src/components/footer/footer.css

```
@@ -0,0 +1,21 @@
+footer {
+  background-color: #031f3d;
+  width: 100vw;
+}
+
+.container-footer {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+}
+
+.container-copyright {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.company-info {
+  width: 30%;

```

como a div pai desses 3 caras é uma div flex, poderia usar `flex: 1` para definir o tamanho delas e não precisar declarar o width aqui:

```suggestion
.container-footer {
  flex: 1;
  display: grid;
  grid-template-columns: 1fr 1fr 1fr;
}

.container-copyright {
  flex: 1;
  display: flex;
  justify-content: center;
  align-items: center;
}
.company-info {
  flex: 1;
```
veja se surte o mesmo efeito

---

não dá o mesmo efeito